starter-code/index.html

```diff
@@ -3,40 +3,65 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600&display=swap" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <header>
+    <img src="images/spotify-logo.png" alt="logo">
+    <ul>
+      <li>Premium</li>
+      <li>Discover</li>
+      <li>Help</li>
+      <li>Download</li>
+    </ul>
+  </header>
+  <section class="eyecatcher">
+    <h1>Music for everyone.</h1>
+    <p>Spotify is now free on mobile, tablet and computer. <br>Listen to the right music, wherever you are.</p>
+  </section>
+  <section class="features">
+    <h2>What's on Spotify?</h2>
+    <div class="container">
+      <div class="container-item">
+        <img src="images/music-icon.png" alt="music-icon">
+        <h3>Millions of Songs</h3>
+        <p>There are millions of songs on Spotify</p>
+      </div>
+      <div class="container-item">
+        <img src="images/high-quality-icon.png" alt="high-quality">
+        <h3>HD Music</h3>
+        <p>Listen to music as if you were listening live</p>
+      </div>
+      <div class="container-item">
+        <img src="images/devices-icon.png" alt="devices">
+        <h3>Stream Everywhere</h3>
+        <p>Stream music on your smartphone, tablet or computer</p>
+      </div>
+    </div>
+  </section>
+  <section class="kanye">
+    <div>
+      <h2>It’s as yeezy as Kanye West.</h2>
+      <div class="kanye-elements">
+        <ul class="kanye-elements-list">
+            <li>
+              <h3>Search</h3>
+              <p>what you want to listen to?</p>
+            </li>
+            <li>
+                <h3>Browse</h3>
+                <p>Check out the latest charts, brand new releases and great playlists for right now.</p>
+            </li>
+            <li>
+                <h3>Discover</h3>
+                <p>Enjoy new music every Monday with your own personal playlist. Or sit back and enjoy Radio.</p>             
+            </li>
+        </ul>
+        <img src="images/spotify-icon-white.png" class="icon">
```

Comment:
That logo might have been easier to handle as a background-image of div.website-functions.